### DISCUSSION TITLE
How to adjust the number of points of the synthesis light curve of the 'emcee_posts' model?

### DISCUSSION THREAD

**Discussion by UltraX3D** [phoebe-project/phoebe2#713] [Questions]
Hi, I have run the MCMC and got a Parameter distribution set **'emcee_posts'**, but I don't know how to set the  points of the synthesis light curve. At present, I can only get the synthesis light curve with the same number of calculated phase points as the initial data set. 
For example, I used code like this
**b.add_dataset('lc', times=T1,fluxes=F1,sigmas=S1,dataset='lcB',passband='Johnson:B',compute_phases=phoebe.linspace(0,1,101))** 
when I first added the data set, so I only got a 101-point synthesis light curve after runing MCMC chain.

Could you please tell me how to adjust the number of points of the composite light curve?

**Comment by kecnry** [comment]
This is covered in detail in the [compute times tutorial](http://phoebe-project.org/docs/latest/tutorials/compute_times_phases).  If you're setting this before running MCMC, the [solver times tutorial](http://phoebe-project.org/docs/latest/tutorials/solver_times.ipynb) might also be applicable.  If anything isn't clear, please let us know so we can improve the docs!
